### PR TITLE
Fixed slider in popups, tabs and other initially hidden elements

### PR DESCRIPTION
**Pull request by kilobyte2007**
I found a way to make the slider work in popups and tabs without the need to trigger a resize event. Let me know if you think the commit is good.

### CODE REVIEW

**Comment by kilobyte2007** [comment]
Some more changes.

File: dist/keen-ui.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*!
```
Comment (JosephusPaye) [inline dist/keen-ui.css]: Remove the dist file changes, we'll add those when we publish the release. Also remove the `lib/` file changes.

File: src/UiSlider.vue [hunk 2]
```diff
@@ -316,9 +303,13 @@ export default {
...
         },
 
+        getTrackLength() {
+            return this.$refs.track.offsetWidth;
+        },
+
         dragUpdate(e) {
             const position = e.touches ? e.touches[0].pageX : e.pageX;
-            const relativeValue = (position - this.trackOffset) / this.trackLength;
+            const relativeValue = (position - this.getTrackOffset(this.$refs.track)) / this.getTrackLength();
```
Comment (JosephusPaye) [inline src/UiSlider.vue]: `this.getTrackOffset()` doesn't take a parameter:

```suggestion
            const relativeValue = (position - this.getTrackOffset()) / this.getTrackLength();
```

File: src/UiSlider.vue [hunk 3]
```diff
@@ -263,23 +258,15 @@ export default {
...
-            this.trackLength = this.$refs.track.offsetWidth;
-            this.trackOffset = this.getTrackOffset(this.$refs.track);
-        },
-
         initializeSlider() {
             document.addEventListener('touchend', this.onDragStop);
             document.addEventListener('mouseup', this.onDragStop);
             document.addEventListener('click', this.onExternalClick);
 
             this.$on('window-resize', () => {
```
Comment (JosephusPaye) [inline src/UiSlider.vue]: We can now remove the listener as we were stopping the drag due to the possible size change. The `RespondsToWindowResize` mixin below can also be removed (with the import above as well)

File: .idea/.gitignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```
Comment (JosephusPaye) [inline .idea/.gitignore]: These idea files seem to have slipped in despite your last commit 👀

File: src/UiSlider.vue (hunk 2 above)
Comment (JosephusPaye) [reply src/UiSlider.vue]: Still relevant

**Comment by kilobyte2007** [comment]
I will create a separate clean pull request.

**Pull request closed without merging**